biorbd_optim/plot.py

```diff
@@ -420,7 +422,18 @@ def callback(self):
             while self.pipe.poll():
                 V = self.pipe.recv()
                 self.plot.update_data(V)
-
+                Iterations.save(V)
```

Comment:
wrong place, this inside the main process, it should be done on the other process so it doesn't lose time during optimization